pkg/config/defaults.go

```diff
@@ -26,6 +26,8 @@ import (
 	"k8s.io/utils/pointer"
 )
 
+var ProjectCloneContainerDefaultCpuLimit = resource.MustParse("1000m")
+
```

Comment:
Could you add a doc line on this variable to explain why it's not included in the defaultConfig below? e.g.

```suggestion
// By default, we do not set a CPU limit for the project clone container. However, to ensure the container 
// still works in clusters with limit ranges configured, this quantity is used when a) a limitrange may set 
// a default value for CPU limit and b) the Operator's configuration does not specify a non-default value.
var ProjectCloneContainerDefaultCpuLimit = resource.MustParse("1000m")
```